internal/invite/invite.go

```diff
@@ -19,19 +19,34 @@ package invite
 import (
 	"crypto/rand"
 	"encoding/base64"
+	"time"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 const (
-	// CodeByteLength = 48 is the length of the invite code in bytes
-	CodeByteLength = 48
+	// codeByteLength = 48 is the length of the invite code in bytes
+	codeByteLength = 48
+	// expireIn7Days is the duration for which an invitation is valid - 7 days
+	expireIn7Days = 7 * 24 * time.Hour
 )
 
 // GenerateCode generates a random invite code
 func GenerateCode() string {
-	bytes := make([]byte, CodeByteLength)
+	bytes := make([]byte, codeByteLength)
 	// Generate random bytes
 	if _, err := rand.Read(bytes); err != nil {
 		return ""
 	}
 	return base64.URLEncoding.EncodeToString(bytes)
 }
+
+// GetExpireIn7Days sets the expiration date of the invitation to 7 days from t.Now()
```

Comment:
nit: The function doesn't set the date, but returns a new date
```suggestion
// GetExpireIn7Days returns the expiration date of the invitation 7 days from t.Now()
```

Reply:
thanks!